axonserver/src/main/java/io/axoniq/axonserver/message/command/CommandCache.java

```diff
@@ -9,6 +9,8 @@
 
 package io.axoniq.axonserver.message.command;
 
+import io.axoniq.axonserver.grpc.ContextProvider;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Use Nonnull instead of jetbrains annotation.

Reply:
see: https://www.jetbrains.com/help/idea/nullable-and-notnull-annotations.html#nullable on how to change the default settings in IntelliJ.